repo/blob/storage.go

```diff
@@ -56,9 +56,9 @@ type OutputBuffer interface {
 // Capacity describes the storage capacity and usage of a Volume.
 type Capacity struct {
 	// Size of volume in bytes.
-	SizeB uint64
+	SizeB uint64 `json:"capacity"`
```

Comment:
Printing out `0` size does not seem to make a lot of sense.
Also, what's a good way to handle or convey "unlimited" capacity.
In the unlimited case, does it make sense to have both of these fields have the `omitempty` tag?

Reply:
That's right, thanks for catching that. I think `omitempty` makes sense here–we could argue that if the storage backend "isn't a volume" (doesn't have a limited capacity), then these fields are irrelevant and shouldn't show up.